### PR TITLE
TD ontology fix v2

### PR DESCRIPTION
**Pull request by relu91**
This PR supersedes #1865 by editing the `ttl` files and moving a bunch of properties from AnnotationProperties to DataTypeProperties. Since I think some of the properties could still be considered as AnnotationProperties in the second commit I added the ability to render AnnotationProperties too.

### CODE REVIEW

**Comment by mahdanoura** [comment]
thanks @relu91. The `owl:DatatypeProperty` defined all require both a `domain` and a `range` which is missing for the following concepts: `followsProfile`, `instance` and `model`. Also, in line 149 in the ontology.ttl there is a spelling mistake: `schema:rageIncludes schema:URL`should be `schema:rangeIncludes schema:URL`

File: ontology/td.ttl [hunk 1]
```diff
@@ -156,6 +136,31 @@
...
+    rdfs:label "followsProfile" ;
+    rdfs:comment "Indicates the WoT Profile mechanisms followed by this Thing Description and the corresponding Thing implementation."@en ;
+    schema:domainIncludes :Thing ;
+    rdfs:isDefinedBy <https://www.w3.org/2019/wot/td> .
+
+:baseURI rdf:type owl:DatatypeProperty ;
+    rdfs:label "baseURI" ;
+    rdfs:comment "Define the base URI that is used for all relative URI references throughout a TD document. In TD instances, all relative URIs are resolved relative to the base URI using the algorithm defined in [RFC3986]. base does not affect the URIs used in @context and the IRIs used within Linked Data [LINKED-DATA] graphs that are relevant when semantic processing is applied to TD instances."@en ;
+    schema:domainIncludes :Thing ;
+    schema:rageIncludes schema:URL ;
```
Comment (mahdanoura) [inline ontology/td.ttl:149]: @relu91 syntax issue in: `schema:rageIncludes`

File: ontology/td.ttl [hunk 2]
```diff
@@ -156,6 +136,31 @@
     schema:domainIncludes wotsec:SecurityScheme ;
     rdfs:isDefinedBy <https://www.w3.org/2019/wot/td> .
 
+:followsProfile rdf:type owl:DatatypeProperty ;
+    rdfs:label "followsProfile" ;
+    rdfs:comment "Indicates the WoT Profile mechanisms followed by this Thing Description and the corresponding Thing implementation."@en ;
+    schema:domainIncludes :Thing ;
+    rdfs:isDefinedBy <https://www.w3.org/2019/wot/td> .
```
Comment (mahdanoura) [inline ontology/td.ttl:143]: also requires a `schema:rangeIncludes`

File: ontology/td.ttl [hunk 3]
```diff
@@ -156,6 +136,31 @@
...
+    rdfs:comment "Define the base URI that is used for all relative URI references throughout a TD document. In TD instances, all relative URIs are resolved relative to the base URI using the algorithm defined in [RFC3986]. base does not affect the URIs used in @context and the IRIs used within Linked Data [LINKED-DATA] graphs that are relevant when semantic processing is applied to TD instances."@en ;
+    schema:domainIncludes :Thing ;
+    schema:rageIncludes schema:URL ;
+    rdfs:isDefinedBy <https://www.w3.org/2019/wot/td> .
+
+:instance rdf:type owl:DatatypeProperty ;
+    rdfs:label "instance" ;
+    rdfs:comment "Provides a version identicator of this TD instance."@en ;
+    schema:domainIncludes :Thing ;
+    rdfs:isDefinedBy <https://www.w3.org/2019/wot/td> .
```
Comment (mahdanoura) [inline ontology/td.ttl:156]: also requires a `schema:rangeIncludes`

File: ontology/td.ttl [hunk 4]
```diff
@@ -156,6 +136,31 @@
...
+
+:model rdf:type owl:DatatypeProperty ;
+    rdfs:label "model" ;
+    rdfs:comment "Provides a version indicator of the underlying TM."@en ;
+    schema:domainIncludes :Thing ;
+    rdfs:isDefinedBy <https://www.w3.org/2019/wot/td> .
```
Comment (mahdanoura) [inline ontology/td.ttl:162]: also requires a `schema:rangeIncludes`

**Review by mahdanoura — CHANGES_REQUESTED** [review]
thanks @relu91. The owl:DatatypeProperty defined all require both a domain and a range which is missing for the following concepts: followsProfile, instance and model. Also, in line 149 in the ontology.ttl there is a syntax mistake: schema:rageIncludes schema:URLshould be schema:rangeIncludes schema:URL

**Comment by egekorkan** [comment]
Call of 27.09:
- It is not exactly correct to have `rangeIncludes` with anyURI since `"profile"` can have multiple values.
- We can merge as is but add the `rangeIncludes` with the correct modeling for anyURI or Array of anyURI. @mahdanoura will check how to model that.

**Pull request merged**